app/views/users/service_providers/show.html.slim

```diff
@@ -8,29 +8,41 @@ h1 = t('headings.service_providers.show', name: service_provider.name)
 
 table.horizontal-headers
   tr
-    th Issuer
-    td.issuer = service_provider.issuer
-  tr
-    th Name
-    td.name = service_provider.name
+    th Friendly Name
+    td.friendly_name = service_provider.friendly_name
   tr
     th Description
     td.description = service_provider.description
   tr
-    th Metadata URL
-    td.metadata_url = service_provider.metadata_url
+    th Issuer
+    td.issuer = service_provider.issuer
+  tr
```

Comment:
Hmm, might be worth revisiting the list of SP attributes in our [dev docs](https://pages.18f.gov/identity-dev-docs/#2-configuring-logingov-to-accept-your-sp), see if we're presenting them accurately. Didn't know that about `agency` and `friendly_name` — here we have them as two totally different concepts.

And yes, there should probably be non-null constraint on `agency`, and actually we probably missed a few others that are required as well (see dev docs).